aphrodite/modeling/models/llama.py

```diff
@@ -340,6 +341,11 @@ def forward(
         hidden_states, _ = self.norm(hidden_states, residual)
         return hidden_states
 
+_printed = set()
+def print_once(key, *args, **kwargs):
```

Comment:
We have a `log_once()` function in `aphrodite.common.logger`. You can use that instead of this.